source/resource/designs/metal/basic.css

```diff
@@ -587,7 +587,11 @@ div#demo_3 {
 	width: auto;
 }
 
-#navbarTop {
+#navbarTop,
+#navbarBottom,
+#navbarLeft,
+#navbarRight
+{
```

Comment:
This changes the left/right navbar background to the gradient. Left/Right navbars in metal design do not have a gradient background only the top/bottom ones.